Lib/test/test_winconsoleio.py

```diff
@@ -68,6 +69,18 @@ def test_open_fd(self):
             f.close()
             f.close()
 
+        with warnings.catch_warnings():
+            warnings.simplefilter("error", RuntimeWarning)
+            try:
+                with self.assertRaises(RuntimeWarning):
+                    f = ConIO(False)
+                with self.assertRaises(RuntimeWarning):
+                    f = ConIO(True, mode='w')
+            except ValueError:
+                # cannot open console because it's not a real console
+                pass
```

Comment:
I think that it is better to run these tests only after we checked that the code works with corresponding integer argument. Add them in the "else" branches above.